src/services/uploader/uploading.controller.test.ts

```diff
@@ -137,4 +145,82 @@ describe('UploadingController', () => {
       expect(upload).toBeTruthy()
     });
   });
+
+  describe('Start encode', () => {
+    it('Should encode successfully', async () => {
+      const jwtToken = 'test_jwt_token';
+
+      await linkedAccountsRepository.linkHiveAccount('singleton/starkerz/hive', 'starkerz');
+
+      const upload = await uploadingService.createUpload({ sub: 'singleton/starkerz/hive', username: 'starkerz', id: 'test_id' });
+
+      process.env.DELEGATED_ACCOUNT = 'threespeak'
```

Comment:
this is just so our authentication doesn't fail, we can change it to `process.env.DELEGATED_ACCOUNT = 'threespeak.beta'` but otherwise the tests will just fail

Reply:
ok no need to change then